src/Microsoft.DotNet.XHarness.Android/AdbRunner.cs

```diff
@@ -97,7 +97,7 @@ public void DumpAdbLog(string outputFilePath, string filterSpec = "")
             // Workaround: Doesn't seem to have a flush() function and sometimes it doesn't have the full log on emulators.
             Thread.Sleep(3000);
 
-            var result = RunAdbCommand($"logcat -d {filterSpec}", TimeSpan.FromMinutes(2));
+            ProcessExecutionResults? result = RunAdbCommand($"logcat -d {filterSpec}", TimeSpan.FromMinutes(2));

```

Comment:
I don't like replacing var with the type here and other places, can we turn that off?

Reply:
I do prefer 'var' but approving nevertheless

Reply:
Yup. I was just doing what `dotnet/runtime` was doing. But I guess we can diverge, no problem